extension/parquet/CMakeLists.txt

```diff
@@ -100,6 +100,7 @@ build_static_extension(parquet ${PARQUET_EXTENSION_FILES})
 set(PARAMETERS "-warnings")
 build_loadable_extension(parquet ${PARAMETERS} ${PARQUET_EXTENSION_FILES})
 target_link_libraries(parquet_loadable_extension duckdb_mbedtls)
+add_definitions(-DZSTD_ASAN_DONT_POISON_WORKSPACE)
```

Comment:
Do we need this define for releases - or is this only relevant for debug mode? Defines won't be set when e.g. compiling for Python.

Reply:
It should only be relevant to debug mode, without asan set it won't do anything.
When this isn't set it would try to call a symbol that wasn't there, maybe I'll just remove those paths instead of setting this define